srcpkgs/scanmem/patches/fix-musl-test.patch

```diff
@@ -0,0 +1,41 @@
+From e5781feb4403e7f2de19f7c027cda5e5ef092a5c Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens@artixlinux.org>
+Date: Fri, 1 Jan 2021 15:46:05 -0600
+Subject: [PATCH] fix test for musl
+
+---
+ test/memfake.c | 8 +++++---
+ 1 file changed, 5 insertions(+), 3 deletions(-)
+
+diff --git test/memfake.c test/memfake.c
+index 1c4fa94..bc2da8e 100644
+--- test/memfake.c
++++ test/memfake.c
+@@ -22,19 +22,21 @@
+ #include <stdlib.h>
+ #include <time.h>
+ #include <unistd.h>
++#include <assert.h>
+ 
+ int main(int argc, char **argv)
+ {
+-    uint MB_to_allocate = 1;
++    size_t MB_to_allocate = 1;
+     bool add_randomness = false;
+ 
+-    if (argc >= 2) MB_to_allocate = atoi(argv[1]);
+-    if (argc >= 3) add_randomness = atoi(argv[2]);
++    if (argc >= 2) MB_to_allocate = strtoul(argv[1], 0, 10);
++    if (argc >= 3) add_randomness = strtoul(argv[2], 0, 10);
+     if (argc >= 4) return 1;
+ 
```

Comment:
What's wrong with this test?

Reply:
Fails on musl because says uint isn’t declared and someone in my Issue upstream recommended it to be written this way, though still waiting for a member of the upstream to comment

Reply:
OK, I see, `atoi(3)` return an `int` instead of `uint`. Change those types to `size_t` is a correct move.
change `add_randomness` to `strtoul` is harmless but not necessary.